autoconnect/autoconnect-web/src/lib.rs

```diff
@@ -17,6 +17,9 @@ use actix_web::web;
 #[macro_export]
 macro_rules! build_app {
     ($app_state: expr) => {
```

Comment:
This kinda feels 'clever' to me, instead of just updating the `test` instance of `build_app!` to include `crate::config()` 
We're already modifying a bunch of similar code already.

Reply:
That's exactly why I did it, so I didn't have to change it anywhere else: but good call, it's only 2 places